rokh/events/gregorian.py

```diff
@@ -164,102 +204,162 @@
+        '15': [{'description': 'درگذشت وندل مردیت استنلی بیوشیمیست و ویروس‌شناس آمریکایی، برنده جایزه نوبل شیمی سال ۱۹۴۷', 'is_holiday': False},
+               {'description': 'روز جهانی آگاهی از بدرفتاری با سالمندان', 'is_holiday': False}],
         '16': [{'description': 'روز جهانی پدر', 'is_holiday': False},
-               {'description': 'زادروز باربارا مک‌کلینتوک دانشمند آمریکایی، برندهٔ جایزه نوبل پزشکی سال ۱۹۸۳', 'is_holiday': False}],
-        '17': [{'description': 'روز جهانی بیابان زدایی', 'is_holiday': False}],
+               {'description': 'زادروز باربارا مک‌کلینتوک دانشمند آمریکایی، برندهٔ جایزه نوبل پزشکی سال ۱۹۸۳', 'is_holiday': False},
+               {'description': 'روز جهانی کمک های مالی خانواده‌ها', 'is_holiday': False}],
+        '17': [{'description': 'روز جهانی مبارزه با بیابان‌زایی و خشکسالی', 'is_holiday': False}],
         '18': [{'description': 'درگذشت پل کارر شیمیدان سوئیسی، برندهٔ جایزه نوبل شیمی سال ۱۹۳۷', 'is_holiday': False},
-               {'description': 'زادروز چارلز لاورن زیست‌شناس فرانسوی، برندهٔ جایزه نوبل فیزیولوژی و پزشکی در سال ۱۹۰۷', 'is_holiday': False}],
+               {'description': 'زادروز چارلز لاورن زیست‌شناس فرانسوی، برندهٔ جایزه نوبل فیزیولوژی و پزشکی در سال ۱۹۰۷', 'is_holiday': False},
+               {'description': 'روز جهانی تغذیه پایدار', 'is_holiday': False},
+               {'description': 'روز جهانی مقابله با سخنان نفرت‌ انگیز', 'is_holiday': False}],
         '19': [{'description': 'زادروز آنگ سان سو چی فعال سیاسی، مدافع حقوق بشر، برنده جایزه صلح نوبل سال ۱۹۹۱', 'is_holiday': False},
-               {'description': 'درگذشت ویلیام گلدینگ شاعر و رمان‌نویس بریتانیایی، برندهٔ جایزه نوبل ادبیات در ۱۹۸۳', 'is_holiday': False}],
-        '20': [{'description': 'درگذشت ژرژ لومتر کشیش و فیزیکدان بلژیکی', 'is_holiday': False}],
-        '23': [{'description': 'زادروز آلن تورینگ، ریاضی‌دان، منطق‌دان، و رمزنگار بریتانیایی، دانشمند علوم رایانه', 'is_holiday': False}],
+               {'description': 'درگذشت ویلیام گلدینگ شاعر و رمان‌نویس بریتانیایی، برندهٔ جایزه نوبل ادبیات در ۱۹۸۳', 'is_holiday': False},
+               {'description': 'روز جهانی مقابله با خشونت جنسی در جنگ‌ها', 'is_holiday': False}],
+        '20': [{'description': 'درگذشت ژرژ لومتر کشیش و فیزیکدان بلژیکی', 'is_holiday': False},
+               {'description': 'روز جهانی پناهندگان', 'is_holiday': False}],
+        '21': [{'description': 'روز جهانی یوگا', 'is_holiday': False},
+               {'description': 'روز جهانی بزرگداشت انقلاب تابستانی (سولستیس)', 'is_holiday': False}],
+        '23': [{'description': 'زادروز آلن تورینگ، ریاضی‌دان، منطق‌دان، و رمزنگار بریتانیایی، دانشمند علوم رایانه', 'is_holiday': False},
+               {'description': 'روز خدمات عمومی سازمان ملل', 'is_holiday': False},
+               {'description': 'روز جهانی بیوه‌ها', 'is_holiday': False},
+               {'description': 'روز جهانی زنان در دیپلماسی', 'is_holiday': False}],
```

Comment:
[International Day of Women in Diplomacy ](https://www.un.org/en/observances/women-in-diplomacy-day)([A/RES/76/269](https://undocs.org/en/A/RES/76/269)) -->‌ 24 Jun